### ISSUE TITLE
Clarify generation time use (default vs model specific?)

### ISSUE THREAD

**Issue by hyanwong** [popsim-consortium/stdpopsim#396]
At the top of the [catalog for H sap](https://stdpopsim.readthedocs.io/en/latest/catalog.html#sec_catalog_HomSap), the docs say ```Generation time: 30 (Tremblay and Vezina, 2000)```, so on first reading, I assumed that the models below used this as a generation time. It's only after more detailed examination of the models that I realised this is something like a *default* Generation time, and specific models define their own generation times (e.g. most older ones for _H. sapiens_ seem to use 25). Should the docs be adjusted to clarify what the Generation time for a species actually does? It's not at all clear to me what the point of the number 30 is, or how it is used downstream (something to do with https://github.com/popsim-consortium/stdpopsim/issues/75 perhaps) 

Confusingly for a novice, in the `OutOfAfrica_3G09` model, and presumably others too, the generation time of 25 is a scaling factor that should simply cancel in the parameter estimates. That means it is perfectly logical to take this model, with its generation time of 25 years, get a TS, and output the TS with a generation time of 30 years. What would be wrong would be to take the `OutOfAfrica_3G09` model and adjust the generation time by hand to 30 from 25. It might be helpful to note this somewhere (I'm happy to have a go, if there is agreement on where such discussion should be placed in the docs).

Possibly another issue, but the real scaling factor in the parameter estimates for this (and probably many other) models is the calibration of the mutation rate using a chimp/human divergence of 6Ma. However, I guess the best we can do to clarify this is what we do already, and provide a link to the original paper from where the user can figure it out the source of the scaling for themselves.

**Comment by gregorgorjanc** [comment]
@hyanwong see how we handled this for model-specific mutation rate in https://github.com/popsim-consortium/stdpopsim/pull/839 and model/specific recombination rate in https://github.com/popsim-consortium/stdpopsim/issues/1225
